test/test_controller.py

```diff
@@ -200,7 +201,7 @@ def test_post_respondent_creates_the_enrolment(self, _):
         self.assertEqual(str(enrolment.business_respondent.business.party_uuid),
                          '3b136c4b-7a14-4904-9e01-13364dd7b972')
 
-    @patch('ras_party.controllers.controller.notify')
+    @patch('ras_party.controllers.controller._send_message_to_gov_uk_notify')
```

Comment:
Maybe one for our tech debt list, would be good to avoid patching something within the unit-under-test, and to patch at the level of the GovUKNotify class instead (I know that it was that way before :)). Uncomfortable about seeing a patch of a "protected" name.